ui/css/treeherder-navbar.css

```diff
@@ -379,6 +379,17 @@ fieldset[disabled] .btn-view-nav-closed.active {
   cursor: default;
 }
 
+#notification-dropdown {
+  width: 40em;
```

Comment:
I wonder if width is needed? Or could be something like `min-width: 12em;` or something if it is? As the current menu I'm seeing when running the branch seems pretty large.

Branch:
![notificationsmenu](https://user-images.githubusercontent.com/3660661/30502699-6085d40a-9a35-11e7-802b-1a675397790a.jpg)

vs.
![notificationsmenu10em](https://user-images.githubusercontent.com/3660661/30502735-7a0c2938-9a35-11e7-9f2a-81f8afcad74d.jpg)

Reply:
agreed

Reply:
I would maybe try right-aligning the Actions items. Would be less mouse movement from opening the list of notifications down to the Clear button.

Reply:
@KWierso curious, do we do right-aligned or blended left/right aligned navbar menus elsewhere? I can't think of any. If it's an action that is that frequent, to avoid the extra pixels mouse movement, we could always make a keyboard shortcut for it. :)

Reply:
@tojon The Filters dropdown puts common actions as buttons in the top-right corner of the dropdown. 

Maybe we could get rid of the "Actions" section of this new dropdown entirely, and just put a "Clear all" button up into the dropdown's main header, next to "Recent Notifications"? I'm not sure what other "Actions" we would have for the notifications, so I don't know it warrants putting that whole section in there.